### PR TITLE
feat: Adds the ability of the rich text renderer to render KvButtons

### PR DESCRIPTION
**Pull request by eddieferrer**
Test page: https://www.dev.kiva.org/lp/eddie-test-landing

- Renders kvButton components from embedded or inline button entries
- Wraps rich text output in kv-tailwind class to fix rendering

SUBS-789

### CODE REVIEW

File: src/util/contentful/richTextRenderer.js [hunk 1]
```diff
@@ -10,50 +10,77 @@ import { BLOCKS, INLINES } from '~/@contentful/rich-text-types';
...
+				<div class="tw-prose">${richTextRenderer(contentfulEntryNode?.data?.target?.fields?.richText)}</div>
+			`;
+		}
+		if (isButton) {
+			const analytics = contentfulEntryNode?.data?.target?.fields?.analyticsClickEvent ?? [];
+			return `
+				<kv-button
+					variant="${contentfulEntryNode?.data?.target?.fields?.style ?? 'primary'}"
+					href="${contentfulEntryNode?.data?.target?.fields?.webLink ?? '#'}"
+					v-kv-track-event="['${analytics.join("','")}']"
```
Comment (eddieferrer) [inline src/util/contentful/richTextRenderer.js:70]: This is not working, I'm looking into it

Comment (mcstover) [reply src/util/contentful/richTextRenderer.js:70]: Interesting. Do you get the correct markup here?

**Review by mcstover — APPROVED** [review]
Another great step forward!

File: src/util/contentful/richTextRenderer.js (hunk 1 above)
Comment (eddieferrer) [reply src/util/contentful/richTextRenderer.js:70]: It is working, made some slight modifications so we arent firing the kv-track-event directive when the field does not exist in contentful

**Pull request merged**